spring-boot-admin-server-ui/src/main/frontend/services/instance.js

```diff
@@ -150,7 +166,7 @@
 
   async clearCache(name, cacheManager) {
     return this.axios.delete(uri`actuator/caches/${name}`, {
-      params: {'cacheManager': cacheManager}
+      params: { cacheManager: cacheManager },
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/ulischulte/spring-boot-admin/pullRequest?prid=8294305)
```suggestion
      params: { cacheManager },
```